### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by nicolets**
now the cart has the number of items of current cart

### CODE REVIEW

**Comment by EvangeliaSp** [comment]
@nicolets There are conflicts. Merge the master in Cart branch, resolve the conflicts and push the changes :)

**Comment by EvangeliaSp** [comment]
@nicolets In the home page I don't see the cart on the top right!

File: frontend/src/App.js [hunk 1]
```diff
@@ -17,35 +15,130 @@ class App extends Component {
     }
 }
 
+class Header extends React.Component {
```
Comment (knikamo) [inline frontend/src/App.js]: I think the header component should be in it's own file and not in `App.js`

File: frontend/src/App.js [hunk 2]
```diff
@@ -17,35 +15,130 @@ class App extends Component {
...
+
+                        allowTransparency="true"></iframe>
 
                 </div>
             </div>
         );
     }
 }
 
+class Footer extends Component {
```
Comment (knikamo) [inline frontend/src/App.js]: same with footer

File: src/main/java/se/uu/elephas/services/OrderServiceImpl.java [hunk 3]
```diff
@@ -138,4 +140,14 @@ public Order proceedOrder(Long idUser) {
 
     }
 
+    public int sizeCart(Long idUser){
+        Order basket=getBasketOfUser(idUser).iterator().next();
+        Iterable<OrderItem> cart =getOrderItemsByOrderId(basket.getIdOrder());
+        int mycounter=0;
+        for (OrderItem item:cart){
+            mycounter = mycounter + 1*item.getQuantity();
```
Comment (knikamo) [inline src/main/java/se/uu/elephas/services/OrderServiceImpl.java]: is it neccessary to multiply with 1?

Comment (EvangeliaSp) [reply src/main/java/se/uu/elephas/services/OrderServiceImpl.java]: haha you are right it was my idea ahahahhah
:1st_place_medal:

**Pull request closed without merging**